src/Faithlife.WebRequests/Json/JsonResponseUtility.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Text.RegularExpressions;
+
+namespace Faithlife.WebRequests.Json
+{
+	internal static class JsonResponseUtility
+	{
+		/// <summary>
+		/// Returns true if the content type is a JSON content type.
+		/// </summary>
+		/// <param name="contentType">The response.</param>
+		/// <returns>True if the content type is "application/json" or "application/schema+json".</returns>
+		public static bool IsJsonContentType(string contentType)
+		{
+			if (contentType == null)
+				throw new ArgumentNullException(nameof(contentType));
+
+			// Length comparison is to short-circuit more-expensive regex check. Could potentially call Trim() and check StartsWith "application" as another performance optimization.
```

Comment:
This seems like a premature optimization. The regex should be fine, and if it isn't, we should get rid of it altogether.

Reply:
I agree that it seems like a premature optimization; I was trying to respect the apparent optimization in the previous code, which was:

```
contentType.Length >= JsonWebServiceContent.JsonContentType.Length &&
				contentType.Trim().StartsWith(JsonWebServiceContent.JsonContentType, StringComparison.Ordinal)
```

I was assuming that the length check was an optimization to avoid calling `StartsWith`, although I guess I'm not really sure that that was the intent. Anyway, I deleted the length check.

Reply:
Good point. The old code looks like a premature optimization as well. Thanks!